packages/front-end/components/Queries/ExpandableQuery.tsx

```diff
@@ -162,6 +162,15 @@ const ExpandableQuery: FC<{
                   )}
                 </strong>
               </div>
+              <div className="col-auto mb-2">
+                <em>Time queued</em>:{" "}
+                <strong>
+                  {formatDistanceStrict(
+                    getValidDate(query.createdAt),
+                    getValidDate(query.startedAt)
+                  )}
+                </strong>
+              </div>
```

Comment:
This is definitely an optional addition but I found myself wanting to check the queued time after re-running the queries so I thought I'd toss it in here. Let me know whether this seems useful to keep or if I should get rid of it.

Reply:
Oh no, thanks for adding this! It will be 0 for most queries, right? Do we only want to show it if the time queued is > some tiny number of `ms`? Fine either way, just a thought.